cogs/Local.py

```diff
@@ -0,0 +1,89 @@
+import discord
+import random
+import os
+from sys import argv
+from dotenv import load_dotenv
+import asyncio
+import urllib
+import urllib.request
+import urllib.error
+import sys, traceback
+import re
+import json
+from discord.ext import commands
+
+class Local(commands.Cog):

```

Comment:
What is "local" about this module? Are these commands that are only relevant for specific servers? What differentiates the functions in this module compared to `memes.py`?

Reply:
i merged the memes and local py file to minimize